.gitignore

```diff
@@ -0,0 +1,7 @@
+# Editor directories and files
+.idea
+.vscode
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
```

Comment:
Thanks for your PR, @gilcierweb!
Editor configs should go to your local `~/.gitignore_global` file (there's an example [here](https://github.com/lucascaton/dotfiles/blob/master/files/gitignore_global#L5-L7)).

Would you mind removing this file, please?